src/components/Navigation/components/Item/tests/Item.test.tsx

```diff
@@ -254,6 +254,20 @@ describe('<Nav.Item />', () => {
       });
     });
 
+    it('delegates external to <UnstyledLink />', () => {
+      const item = mountWithNavigationProvider(
+        <Item label="some label" url="foo" external disabled={false} />,
+        {
+          location: 'bar',
+        },
+      );
+
+      expect(item).toContainReactComponent(UnstyledLink, {
+        url: 'foo',
+        external: true,
+      });
+    });
+
```

Comment:
We should add a test to make sure the icon gets rendered as well

Something like:

```jsx
expect(item).toContainReactComponent(Icon, {
  accessibilityLabel: en.Polaris.Common.newWindowAccessibilityHint,
  source: ExternalMinor,
});
```

Then you'll need some additional imports:
```jsx
import en from '../../../../../../locales/en.json';
import {PlusMinor, ExternalMinor} from '@shopify/polaris-icons';
```

Reply:
Will do! No worries at all, thanks so much for catching that!